openslides/motions/static/js/motions/base.js

```diff
@@ -385,9 +386,27 @@ angular.module('OpenSlidesApp.motions', [
                 getSearchResultName: function () {
                     return this.getTitle();
                 },
-                // subtitle of search result
-                getSearchResultSubtitle: function () {
-                    return "Motion";
+                // return true if a specific relation matches for given searchquery
+                // e.g. submitter, supporters or category
+                hasSearchResult: function (results, searchquery) {
+                    var motion = this;
+                    var foundSomething;
+                    // search for submitters and supporters (check if any user.id from already found users matches)
+                    _.forEach(results, function(result) {
+                        switch(result.getResourceName()) {
+                            case "users/user":
+                                if (_.find(motion.submitters, {'id': result.id})) {
+                                    foundSomething = true;
+                                } else if (_.find(motion.supporters, { 'id': result.id })) {
+                                    foundSomething = true;
+                                }
+                        }
+                    });
+                    // search for category
+                    if (motion.category && motion.category.name.match(new RegExp(searchquery, 'i'))) {
```

Comment:
The resource `motions/category` is not a registed search module (because only base apps should be registed and add the app name as search filter). So a new switch/case does not work. - The easiest way is to search the categories manually. A more generic solution is not really required because it is only important for motion category.